### PR TITLE
Run Cirrus only for labeled PRs or tags

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Review by samspills — APPROVED** [review]
🙌

**Review by djspiewak — DISMISSED** [review]
Can we also make them run unconditionally for release tags? Also we should probably create the labels. :-)

File: .cirrus.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 jvm_highcore_task:
+  only_if: $CIRRUS_TAG != '' || $CIRRUS_PR != ''
```
Comment (armanbilge) [inline .cirrus.yml:2]: @djspiewak they already run unconditionally for release tags. Unless I'm missing your point.

Comment (djspiewak) [reply .cirrus.yml:2]: I misunderstood the diff. lgtm then

**Pull request merged**